specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzetext.json

```diff
@@ -837,32 +837,32 @@
           "type": "string",
           "description": "Healthcare Entity Category.",
           "enum": [
-            "BODY_STRUCTURE",
-            "AGE",
-            "GENDER",
-            "EXAMINATION_NAME",
-            "DATE",
-            "DIRECTION",
-            "FREQUENCY",
-            "MEASUREMENT_VALUE",
-            "MEASUREMENT_UNIT",
-            "RELATIONAL_OPERATOR",
-            "TIME",
-            "GENE_OR_PROTEIN",
-            "VARIANT",
-            "ADMINISTRATIVE_EVENT",
-            "CARE_ENVIRONMENT",
-            "HEALTHCARE_PROFESSION",
-            "DIAGNOSIS",
-            "SYMPTOM_OR_SIGN",
-            "CONDITION_QUALIFIER",
-            "MEDICATION_CLASS",
-            "MEDICATION_NAME",
-            "DOSAGE",
-            "MEDICATION_FORM",
-            "MEDICATION_ROUTE",
-            "FAMILY_RELATION",
-            "TREATMENT_NAME"
+            "BodyStructure",
```

Comment:
Just to double check, all these were verified with the service, right? It wasn't just a case-change and assuming they were correct? (I mean, hopefully that's all it is, but want to be sure.)

Reply:
Checked the response on May, July (unified language) as well as v3.1 (text analytics) in Japan prod and confirming that the returned category was a pascal case.

Reply:
Does this mean that the original version was never tested to ensure the correct values? This is very disturbing!